Documentation/toc.yml

```diff
@@ -140,6 +140,8 @@
     items:
     - name: Build Window
       href: Utilities/BuildWindow.md
+    - name: Build Window
```

Comment:
Should this be?

`- name: Dependency Window`

Reply:
fixed